srcpkgs/python3-gobject/template

```diff
@@ -16,6 +16,7 @@ license="LGPL-2.1-or-later"
 homepage="https://pygobject.readthedocs.io/"
 distfiles="${GNOME_SITE}/pygobject/${version%.*}/pygobject-${version}.tar.xz"
 checksum=9b12616e32cfc792f9dc841d9c472a41a35b85ba67d3a6eb427e307a6fe4367b
+make_check=no # https://gist.github.com/27c7c0774329deeac7fbe3d7f10ea12c
```

Comment:
This is an error log from glib-networking and this commit (8f407482e70b64d4a0b15c1d832ae9ed50569b3b) can be dropped. I already pushed a python3-gobject update to master, but it wasn't skipped during a rebase because of this one different line.